doc/source/release/1.15.0-notes.rst

```diff
@@ -17,96 +17,1442 @@ run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
+these features by setting an environment variable ``SCIPY_ARRAY_API=1`` and
+providing PyTorch, JAX, ndonnx, or CuPy arrays as array arguments. Features
+with support added for SciPy ``1.15.0`` include:
+
+- All functions in `scipy.differentiate` (new sub-package)
+- All functions in `scipy.optimize.elementwise` (new namespace)
+- `scipy.optimize.rosen`, `scipy.optimize.rosen_der`, and
+  `scipy.optimize.rosen_hess`
+- `scipy.special.logsumexp`
+- `scipy.integrate.trapezoid`
+- `scipy.integrate.tanhsinh` (newly public function)
+- `scipy.integrate.cubature` (new function)
+- `scipy.integrate.nsum` (new function)
+- `scipy.special.chdtr`, `scipy.special.betainc`, and `scipy.special.betaincc`
+- `scipy.stats.boxcox_llf`
+- `scipy.stats.differential_entropy`
+- `scipy.stats.zmap`, `scipy.stats.zscore`, and `scipy.stats.gzscore`
+- `scipy.stats.tmean`, `scipy.stats.tvar`, `scipy.stats.tstd`,
+  `scipy.stats.tsem`, `scipy.stats.tmin`, and `scipy.stats.tmax`
+- `scipy.stats.gmean`, `scipy.stats.hmean` and `scipy.stats.pmean`
+- `scipy.stats.combine_pvalues`
+- `scipy.stats.ttest_ind`, `scipy.stats.ttest_rel`
+- `scipy.stats.directional_stats`
+- `scipy.ndimage` functions will now delegate to ``cupyx.scipy.ndimage``,
+  and for other backends will transit via NumPy arrays on the host.
+
+
+*******************
+Deprecated features
+*******************
+- Functions `scipy.linalg.interpolative.rand` and
+  `scipy.linalg.interpolative.seed` have been deprecated and will be removed
+  in SciPy ``1.17.0``.
+- Complex inputs to `scipy.spatial.distance.cosine` and
+  `scipy.spatial.distance.correlation` have been deprecated and will raise
+  an error in SciPy ``1.17.0``.
+- `scipy.spatial.distance.kulczynski1` and
+  `scipy.spatial.distance.sokalmichener` were deprecated and will be removed
+  in SciPy ``1.17.0``.
+- `scipy.stats.find_repeats` is deprecated as of SciPy ``1.15.0`` and will be
+  removed in SciPy ``1.17.0``. Please use
+  ``numpy.unique``/``numpy.unique_counts`` instead.
+- `scipy.linalg.kron` is deprecated in favour of ``numpy.kron``.
+- Using object arrays and longdouble arrays in `scipy.signal`
+  convolution/correlation functions (`scipy.signal.correlate`,
+  `scipy.signal.convolve` and `scipy.signal.choose_conv_method`) and
+  filtering functions (`scipy.signal.lfilter`, `scipy.signal.sosfilt`) has
+  been deprecated as of SciPy ``1.15.0`` and will be removed in SciPy
+  ``1.17.0``.
+- `scipy.stats.linregress` has deprecated one-argument use; the two
+  variables must be specified as separate arguments.
+- ``scipy.stats.trapz`` is deprecated in favor of `scipy.stats.trapezoid`.
+- `scipy.special.lpn` is deprecated in favor of `scipy.special.legendre_p_all`.
+- `scipy.special.lpmn` and `scipy.special.clpmn` are deprecated in favor of
+  `scipy.special.assoc_legendre_p_all`.
+- The raveling of multi-dimensional input by `scipy.linalg.toeplitz` has
+  been deprecated. It will support batching in SciPy ``1.17.0``.
```

Comment:
```suggestion
  been deprecated. It will support batching in SciPy ``1.17.0``.
  - The ``random_state`` and ``permutations`` arguments of `scipy.stats.ttest_ind`
     are deprecated. Use ``method`` to perform a permutation test.
```